test/server/hot_restarting_parent_test.cc

```diff
@@ -16,17 +17,25 @@ namespace {
 
 using HotRestartMessage = envoy::HotRestartMessage;
 
+int64_t timeToSeconds(MonotonicTime monotonic_time) {
+  return std::chrono::duration_cast<std::chrono::seconds>(monotonic_time.time_since_epoch())
```

Comment:
maybe we should move this to a helper method in include/envoy/common/time.h as you have this form elsewhere?